docs/usage/quickstart.rst

```diff
@@ -0,0 +1,119 @@
+.. _usage/quickstart:
+
+Getting started
+===============
+
+After going through the installation process (see :ref:`usage/installation`),
+the ``lumi`` CLI tool should be at your disposal. This tool is the main way to
+interact with Luminoth, allowing you to train new models, evaluate them, use
+them for predictions, manage your checkpoints and more. Running it will provide
+additional information::
+
+  Usage: lumi [OPTIONS] COMMAND [ARGS]...
+
+  Options:
+    -h, --help  Show this message and exit.
+
+  Commands:
+    checkpoint  Groups of commands to manage checkpoints
+    cloud       Groups of commands to train models in the...
+    dataset     Groups of commands to manage datasets
+    eval        Evaluate trained (or training) models
+    predict     Obtain a model's predictions.
+    server      Groups of commands to serve models
+    train       Train models
+
+We'll start by downloading a checkpoint. Luminoth provides already-trained
+models so you can run predictions and get reasonable results in no time (and
+eventually be able to use for fine-tuning). In order to access these
+checkpoints, we first need to download the remote index with the available
+models.
+
+Checkpoint management is handled by the ``lumi checkpoint`` subcommand. Run the
+following to both retrieve and list the existing checkpoints::
+
+  $ lumi checkpoint refresh
+  Retrieving remote index... done.
+  2 new remote checkpoints added.
+  $ lumi checkpoint list
+  ================================================================================
+  |           id |                  name |       alias | source |         status |
+  ================================================================================
+  | 48ed2350f5b2 |   Faster R-CNN w/COCO |    accurate | remote | NOT_DOWNLOADED |
+  | e3256ffb7e29 |      SSD w/Pascal VOC |        fast |  local | NOT_DOWNLOADED |
+  ================================================================================
+
+Two checkpoints are present:
+
+- **Faster R-CNN w/COCO** (48ed2350f5b2): object detection model trained on the
+  Faster R-CNN model using the COCO dataset. Aliased as ``accurate``, as it's
+  the slower but more accurate detection model.
+
+- **SSD w/Pascal VOC** (e3256ffb7e29): object detection model trained on the
+  Single Shot Multibox Detector (SSD) model using the Pascal dataset. Aliased
+  as ``fast``, as it's the faster but less accurate detection model.
+
+Additional commands are available for managing checkpoints, including inspection
+and modification of checkpoints (see :ref:`cli/checkpoint`).  For now, we'll
+download a checkpoint and use it::
+
+  $ lumi checkpoint download 48ed2350f5b2
+  Downloading checkpoint...  [####################################]  100%
+  Importing checkpoint... done.
+  Checkpoint imported successfully.
+
+Once the checkpoint is downloaded, it can be used for predictions. There are
```

Comment:
Maybe mention where the thing is stored once downloaded?

Reply:
I wouldn't add it to the quickstart section, but it should be mentioned on the checkpoint one.